### ISSUE TITLE
run_MultiPEN.sh file

### ISSUE THREAD

**Issue by dvpolyakov** [EarlhamInst/MultiPEN#3]
Should user launch file run_MultiPEN.sh before MultiPEN file? There are no information about usage of run_MultiPEN.sh file in the guidance. But, as I understand,  this file is important in the process of PATH setting.

**Comment by wjurkowski** [comment]
that's right, this is the one. Please feel free to correct documentation accordingly
